bbc/simorgh
# Remove `isCaf` flag in favour of `CafEnabledServices` array

Overall changes
======
Removes the `isCaf` flag from the `RequestContext` as it is no longer in use.

We have a global `CafServices` array that is used for checking if the service is enabled to fetch/render under CAF

Code changes
======
- Removes `isCaf` flag setting from `server/index` and `RequestContext`
- Removes last instances of `caftest` and `caflive` `renderer_env` flags
- Replaces instances of `isCaf` flag with checking if the service being rendered is in the `CafEnabledServices` array

Testing
======
1. Visit http://localhost:7080/thai/61719672?renderer_env=live
2. Confirm it renders in the ArticlePage component

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
